util/drakeUtil.h

```diff
@@ -8,6 +8,7 @@
 #include "mex.h"
 #include <vector>
 #include <utility>
+#include <dlfcn.h>
```

Comment:
I left it in in case we wanted a way to just call a mex file in a one-off way.  I'd be ok with zapping it since MexWrapper subsumes all of its functionality.  It looks like Windows doesn't like dlfcn.h.  I was always accustomed to using LoadLibrary and friends to do dynamic library loading on Windows.  

Do you think it would be worth a heavily conditional compile off MexWrapper to make it work on windows?  I feel like excluding drakeUtil from compiling on windows is overkill.